packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -91,6 +92,11 @@ class GoogleMap extends StatefulWidget {
   /// True if the map view should respond to zoom gestures.
   final bool zoomGesturesEnabled;
 
+  /// True if the map view should be in lite mode. Android only.
+  ///
+  /// See https://developers.google.com/maps/documentation/android-sdk/lite#overview_of_lite_mode for more details.
```

Comment:
Maybe also explain how to achieve this on iOS. (link to a location where static map api is documented, for example)

Reply:
@cyanglaz it is different API and this is not for free